src/plugin/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -467,14 +465,13 @@ var Foo = function () {
       return _f;
     }()
   }]);
-  return Foo;
```

Comment:
wait what?

Reply:
it was because the babel plugin version changed i assume